js/views/itemEditVw.js

```diff
@@ -143,7 +143,7 @@ module.exports = Backbone.View.extend({
 
   changeType: function(e) {
     var typeValue = $(e.target).val();
-    if(typeValue === "physical good") {
+    if(typeValue === "physical good") { //TODO: remove magic text, make constant or lookup
```

Comment:
In this case, the value is what's set by the API, we don't need to over-optimize here.